### PR TITLE
Signup and Login Front-End Portion

### LINKED ISSUE #3: FE: Signup / Login Pages
- create two front-end react routers (login and signup)
- There is no landing page - redirect to signup
- Signup page - name, email, password + confirm password
- Ignore Forget password (we don't need that right now)
- Front-end validation as well:
    - Examples: Password length >= 6, All fields are required, email is properly formatted (if you wrap signup and login in an HTML form, you can leverage some front-end HTML form validation, required and right typing)
    - leverage error, helperText, and snackbars (material-ui) for showing validation errors, https://material-ui.com/components/snackbars/ + https://material-ui.com/components/text-fields/

### PR DESCRIPTION
**Pull request by AltiV**
Preliminary front-end pages for Signup and Login

### CODE REVIEW

File: client/src/pages/Signup.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    margin: "24px auto 32px",
+    textTransform: "none",
+    backgroundColor: "#32BE72",
+    color: "white",
+    width: "120px",
+    height: "40px",
+  },
+}));
+
+export default function Signup() {
```
Comment (mpmpx) [inline client/src/pages/Signup.js]: I saw in some files, you use export default at the bottom of the file. Would you please make it consistent.

File: client/src/pages/Login.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    margin: "24px auto 32px",
+    textTransform: "none",
+    backgroundColor: "#32BE72",
+    color: "white",
+    width: "120px",
+    height: "40px",
+  },
+}));
+
+export default function Login() {
```
Comment (mpmpx) [inline client/src/pages/Login.js]: I saw in some files, you use export default at the bottom of the file. Would you please make it consistent.

**Review by mpmpx — CHANGES_REQUESTED** [review]
Would you please fix inconsistency of function exports in different files. Then everything is ok to merge

File: client/src/pages/Signup.js (hunk 1 above)
Comment (Travmatth) [reply client/src/pages/Signup.js]: My preference would be we use the format he used in `Navbar` as the default:
```
const Nav = () => {
  return ();
};

export default Nav;
```

**Comment by AltiV** [comment]
Fixed inconsistency for signup/login export format. Merging and closing.

File: client/src/themes/theme.js [hunk 3]
```diff
@@ -6,9 +6,20 @@ export const theme = createMuiTheme({
...
+      color: "black",
+    },
+    subtitle1: {
+      fontWeight: "bold",
+    },
   },
   palette: {
-    primary: { main: "#DF1B1B" }
-  }
+    // primary: { main: "#DF1B1B" },
```
Comment (sina-jamshidi) [inline client/src/themes/theme.js:19]: we can remove all commented out code before merging, can you remove this in your next PR.

Comment (AltiV) [reply client/src/themes/theme.js:19]: Will do

**Pull request merged**